src/NATS.Client.Core/Internal/ClientOpts.cs

```diff
@@ -97,23 +97,12 @@ public static ClientOpts Create(NatsOpts opts)
 
     private static string GetAssemblyVersion()
     {
-        var asm = typeof(ClientOpts);
-        var version = "1.0.0";
-        var infoVersion = asm!.GetCustomAttribute<AssemblyInformationalVersionAttribute>();
-        if (infoVersion != null)
-        {
-            version = infoVersion.InformationalVersion;
-        }
-        else
-        {
-            var asmVersion = asm!.GetCustomAttribute<AssemblyVersionAttribute>();
-            if (asmVersion != null)
-            {
-                version = asmVersion.Version;
-            }
-        }
-
-        return version;
+        var asm = typeof(ClientOpts).Assembly;
+
+        return asm?.GetCustomAttribute<AssemblyFileVersionAttribute>()?.Version
+            ?? asm?.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion
+            ?? asm?.GetCustomAttribute<AssemblyVersionAttribute>()?.Version
+            ?? "?.?.?";
```

Comment:
let's keep this as 1.0.0 in case there is something out there parsing it as numbers

Reply:
Have done that, but thinking about it, maybe 2.0.0 might be better as V2 of the library?

Reply:
ah yes! good point